src/style.css

```diff
@@ -1,3 +1,106 @@
 body {
+  font-family: Roboto, Arial, sans-serif;
+  text-transform: uppercase;
+  width: 100%;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.roboto-medium {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
The '.roboto-medium' class duplicates the 'font-family' from the 'body' selector. If 'Roboto' is the main font for the website, there's no need to redeclare it here unless the font is different for '.roboto-medium' class.